tilty/emitters/webhook.py

```diff
@@ -43,18 +43,20 @@ def __init__(self, config: dict) -> None:
         self.delay_minutes: Union[int, None] = delay_minutes
         self.headers: dict = json.loads(config['headers'])
         self.template: Template = Template(config['payload_template'])
-        self.delay_until: Union[datetime.datetime, None] = None
+        self.delay_until: Dict[str, datetime.datetime] = dict()
 
-    def emit(self, tilt_data: dict) -> None:  # pylint:disable=inconsistent-return-statements  # noqa
+    def emit(self, tilt_data: dict) -> None:
         """ Initializer
 
         Args:
             tilt_data (dict): data returned from valid tilt device scan
         """
 
+        tilt_uuid = str(tilt_data['uuid'])
```

Comment:
what about using a config.get('delay_until_identifier', 'uuid') up in the initializer around line 46 ? that way it can be overridden to use mac in case someone has multiples of the same color

Reply:
Hmm... Originally I was just trying to identify the colors, I was assuming they had a 1:1:1 mapping between color, uuid, and mac address. Now I'm wondering if, say, two blue tilts actually have different mac addresses? You might be right in making it an option, in case someone does have two of the same color and can find some unique attribute to tell the difference between them. Yeah, I'll switch it over to a config value after all when I get a free moment 👍.

Reply:
The mac address is 100% unique based on FCC requirements

The UUID is just a generic bit of data in the BLE implementation [here](https://github.com/myoung34/tilty/blob/master/tilty/blescan.py#L65) . Whoever did the ibeacon implementation for the Tilt set it up so that the UUID is 1:1 with the color so that its easy to find in the ibeacon packet.

Reply:
Some more info is here: https://github.com/myoung34/tilty/issues/16

Basically i added mac everywhere on the off chance that someone has more than the soft maximum limit of 8 tilts (by color)

In theory you could have as many as you like if you dont worry about distinguishing by color but instead by mac. the hard part is that there's no way to see the mac address from the tilt device without scannign for it which is a bit 🤮

Reply:
I've always considered mac address as one of those things that if you rely on it being unique, you tend to quickly find out people aren't following requirements.

I can't think of a scenario where someone would want to delay logging by color then, so mac would almost always be the right choice: "each device should wait X minutes", where a device corresponds to a mac... but maybe I'm just not creative enough, so config seems reasonable and people can decide to do color/uuid or mac, or get weird with it and use temp/gravity even.

I can't check right now because my tilts are busy doing their thing, but I think you can read the mac address through the cover.

Reply:
I think of tilt as being for any kind of techy brew person and color being kind of and obvious thing for it. It's first class.

Mac is more niche than that. As much as I'd like to default to mac for everything I feel like that's kind of pushy to the general users. I can confirm you can't read the Mac from the device at all. It's not on any of the chips. Which is why I add support to it but prioritize color because 98% of the people using tilts are going to own less than a handful and know color.

The only people concerned with Mac would be the small percentage that need more than 8